guides/common/modules/con_provisioning-management-with-foreman-additional-resources.adoc

```diff
@@ -0,0 +1,4 @@
+[role="_additional-resources"]
+= Additional resources
```

Comment:
Again, I worry about the uniqueness of this anchor. Maybe go with ".Additional resources" and a content-specific anchor?